sapl/parlamentares/forms.py

```diff
@@ -470,6 +466,30 @@ def save(self, commit=True):
         return frente
 
 
+class FrenteParlamentarForm(ModelForm):
+    logger = logging.getLogger(__name__)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['frente'].widget = forms.HiddenInput()
+
+    class Meta:
+        model = FrenteParlamentar
+        fields = '__all__'
+
+    def clean(self):
+        cd = super().clean()
+        if not self.is_valid():
+            return self.cleaned_data
+
+        if cd['cargo'].cargo_unico:
+            frente_parlamentar = FrenteParlamentar.objects.filter(cargo=cd['cargo'])[0]
```

Comment:
Quando não existe nenhum parlamentar cadastrado no cargo único da frente, ocorre um erro porque a posição zero também não existe.
Se não filtrar pela frente, uma frente não poderá ter o cadastro de um cargo único depois que outra teve. 
```suggestion
            frente_parlamentar = FrenteParlamentar.objects.filter(frente=cd['frente'], cargo=cd['cargo'])
            f = frente_parlamentar[0] if frente_parlamentar.exists() else None
```